Frends.LDAP.SearchObjects/Frends.LDAP.SearchObjects/SearchObjects.cs

```diff
@@ -86,6 +110,8 @@ public static Result SearchObjects([PropertyTab] Input input, [PropertyTab] Conn
         }
         catch (LdapException ex)
         {
+            if (connection.ThrowExceptionOnError)
```

Comment:
We have another catch block below, which needs to be respecting this flag as well

Reply:
I was thinking this in a way that we use that flag only with actual LDAP exceptions and if there's something else wrong, like connection errors, or invalid parameters, then the Task should always throw an exception.